src/Couchbase.Lite.Shared/Support/Android/XEReachability.cs

```diff
@@ -52,11 +53,20 @@ private void OnConnectivityChanged(object sender, ConnectivityChangedEventArgs e
 
         public void Start()
         {
+            if(!Droid.HasReachability) {
+                WriteLog.To.Sync.W("CouchbaseLite", "Xamarin essentials has not been initialized, disabling reachability (be sure to call Xamarin.Essentials.Platform.Init");
```

Comment:
I think you should capitalize "essentials".  Better yet, say "Xamarin.Essentials.Platform"